### DISCUSSION TITLE
本地大模型如何支持插件功能

### DISCUSSION THREAD

**Discussion by george-ray** [lobehub/lobehub#2098] [Self Hosting]
基于 Ollama 在本地部署大模型, 如何支持 LobeChat 的插件 (function calling) 功能?

**Comment by arvinxx** [comment]
本地模型如果支持 Function Calling 的话应该可以直接用的吧

**Reply by george-ray** [reply]
您好! 请问有支持 Functiong Calling 的本地大模型帮推荐一下吗, 我在 Ollama 的模型库中还没找到合适的.
